src/components/home/cobe.jsx

```diff
@@ -0,0 +1,143 @@
+import createGlobe from 'cobe';
+import { useEffect, useRef } from 'react';
+import { useSpring } from 'react-spring';
+
+export default function Cobe() {
+  let phi = 0;
+
+  const canvasRef = useRef(null);
+
+  const pointerInteracting = useRef(null);
+  const pointerInteractionMovement = useRef(0);
+
+  const genrateRandomMarkers = () => {
+    const markers = [];
+
+    for (let i = 0; i < 10000; i++) {
+      markers.push({
+        location: [Math.random() * 180 - 90, Math.random() * 360 - 180],
+        size: Math.random() * 0.1,
+      });
+    }
+
+    return markers;
+  };
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **nitpick (typo):** Typo in function name 'genrateRandomMarkers'; consider renaming to 'generateRandomMarkers'.